src/main/java/com/splitspendings/groupexpensesbackend/model/SpendingComment.java

```diff
@@ -1,20 +1,28 @@
 package com.splitspendings.groupexpensesbackend.model;
 
-import lombok.Getter;
-import lombok.Setter;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 
 import javax.persistence.*;
 import java.time.ZonedDateTime;
 
 @Entity
 @Table(name = "spending_comment")
-@Getter
-@Setter
+@Data
```

Comment:
Remove @Data.
@Data genereates toString(), equals() and hashCode().
If a data class has relations with other tables and one of these methods is called it will fetch all related models from database.
In worst case scenario (if all models are connected and all have @Data) you can fetch you whole database when one of these methods is called on a model object.